File: src/main/java/no/nav/domain/pensjon/regler/BatchStatus.java
package no.nav.domain.pensjon.regler;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/**
 * Beskriver resultatet for en g-omregning.
 * Hvert element i resultatlisten fra en g-omregning har en instans av dette objektet.
 * 
 * @author utvikler
 */
public class BatchStatus implements Serializable {


    private boolean statusOK;

    /**
     * Liste av merknader. Beskriver hvorfor g-omregningen feilet for det gjeldende objektet.
     * Dersom statusOK er true finnes ingen merknader.
     */
    protected List<Merknad> merknadListe;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/BatchStatus.kt
package no.nav.pensjon.regler.domain

import java.io.Serializable

class BatchStatus : Serializable {
    var statusOK = false

    /**
     * Liste av merknader. Beskriver hvorfor g-omregningen feilet for det gjeldende objektet.
     * Dersom statusOK er true finnes ingen merknader.
     */
    var merknadListe: MutableList<Merknad> = mutableListOf()

    val test = 0
}